rafx-assets/src/assets/upload.rs

```diff
@@ -641,7 +641,7 @@ impl UploadManager {
             buffer_upload_result_rx,
             buffer_upload_result_tx,
             astc4x4_supported: false,
-            bc7_supported: true,
+            bc7_supported: false,
```

Comment:
would like to turn this on again, although maybe there is a better way to do it (where the API exposes flags to say if it's supported)